client/state/reader/site-blocks/selectors/get-site-blocks-last-page.js

```diff
@@ -1,15 +1,10 @@
-/**
- * External dependencies
- */
-import { get } from 'lodash';
-
 /**
  * Internal dependencies
  */
 import 'state/reader/init';
 
 export const getSiteBlocksLastPage = state => {
-	return get( state, [ 'reader', 'siteBlocks', 'lastPage' ] );
+	return state?.reader?.siteBlocks?.lastPage;
```

Comment:
Assuming that `reader` reducer has been loaded, `siteBlocks` and its `lastPage` will always exist.  Also, what is the chance that we'll call the selector with a `null` or `undefined` `state`?